README.md

```diff
@@ -57,15 +57,15 @@ Qlik Hackathon 2015 Start.qvf - Uses data from Medair to analyse Ebola Outbreak
 
 This app uses also Qlik Sense Extension VariableButton: https://github.com/asierfer/QlikSense-Ext-VariableButton
 
-![QlikSense Extension 2DimHeatmap](Screenshot3.PNG)
+![QlikSense Extension 2DimHeatmap](resources/Screenshot3.PNG)
```

Comment:
similar files are in examples folder in previous extension (i actually like resources better), we should change one of them to keep the extensions as similar as possible

Reply:
Okay, changing to resources in the other one